From fb787cb32b618cbee865a8ee751aaf88f61993ee Mon Sep 17 00:00:00 2001
From: Alexandre Manhaes Savio <alexsavio@gmail.com>
Date: Tue, 9 Aug 2016 16:00:38 +0200
Subject: [PATCH] fix ifstream error

---
 Examples/antsSurf.cxx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Examples/antsSurf.cxx b/Examples/antsSurf.cxx
index c89e07c..a868731 100644
--- a/Examples/antsSurf.cxx
+++ b/Examples/antsSurf.cxx
@@ -573,7 +573,7 @@ int antsImageToSurface( itk::ants::CommandLineParser *parser )
 
     // Read in color table
 
-    std::ifstream fileStr( lookupTableFile );
+    std::ifstream fileStr( lookupTableFile.c_str() );
     if( !fileStr.is_open() )
       {
       std::cerr << " Could not open file " << lookupTableFile << '\n';
-- 
2.7.4

